administrator/language/de-DE/com_joomlaupdate.ini

```
@@ -137,7 +137,7 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Die Warnungen über
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NOUPDATES="Keine Updates verfügbar"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NOUPDATESNOTICE="Es ist bereits die aktuellste Joomla-Version %s installiert."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE="Update-Paket-URL"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_INFO="Das <a href='%s' class='alert-link' target='_blank' rel='noopener noreferrer'>Update-Paket</a> kann auch heruntergeladen und manuell installiert werden."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_INFO="Das <a href='%s' class='alert-link' target='_blank' rel='noopener noreferrer'>Update-Paket</a> kann auch auf den Computer heruntergeladen und dann mit dem unten stehenden Feld hochgeladen und installiert werden."
```

Mein Vorschlag wäre zwar keine direkte Übersetzung, klingt für mich aber etwas "geschmeidiger".

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_INFO="Das <a href='%s' class='alert-link' target='_blank' rel='noopener noreferrer'>Update-Paket</a> kann auch lokal gespeichert und dann über die untere Schaltfläche hochgeladen und installiert werden."
```